recipes/tvb-framework/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = 'tvb-framework' %}
+{% set version = '1.5.5' %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  md5: 3f501c3f43705440f8953bf24aa6b0e5
+  sha256: 81c6b293bff080f4b52ec2c9ece5a3e109c951eb9dd42b1d90aed00589d4828b
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/t/tvb-framework/{{ name }}-{{ version }}.tar.gz
```

Comment:
There is a mix of using jinja var and spelling the name out in this URL. Please choose one.
(See the example folder.)

Reply:
got it